artifact/image/layerscanning/image/layer_test.go

```diff
@@ -584,21 +584,6 @@ func TestChainFSReadDir(t *testing.T) {
 // CHAINLAYER TESTING HELPER METHODS
 // ========================================================
 
-func checkError(t *testing.T, funcName string, gotErr error, wantErr error) {
```

Comment:
@another-rex do we need these for layer extraction?

Reply:
No I don't think so, and I think we would want to use cmp.Diff instead anyway I believe.